pennylane_lightning/src/tests/Test_Cache.cpp

```diff
@@ -0,0 +1,135 @@
+#include <catch2/catch.hpp>
+
+#include "Cache.hpp"
+
+using namespace Pennylane::Util;
+
+namespace {

```

Comment:
The test-case here does not need to be within an anonymous namespace block.

Reply:
I will remove it. Thanks.